content/millennium/r4/financial/general/charge-item.md

```diff
@@ -157,6 +157,65 @@ List an individual ChargeItem by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-credit
+
+<%= beta_tag(action: true) %>
+
+Creates an offsetting ChargeItem for an existing debit ChargeItem.
+
+    POST /ChargeItem/:id/$credit
+    
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.    
+    
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/fhir+json', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+Notes:
+
+* versionId, transactionId, subject and context are the required fields.
+
+<%= definition_table(:charge_item_credit, :credit, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.cerner.com/r4/d242a518-4074-4bd4-a3a6-adfe0c5c1c51/ChargeItem/292427912/$credit
```

Comment:
should we use domain ec2458f2-1e24-41c8-b71b-0e701af7583d for our example? I see that domain used in other POST call examples for RelatedPerson and Appointment.

Reply:
Done: https://github.com/cerner/fhir.cerner.com/pull/726/commits/0695602080d10522b8a9f77e73180c5d96f404eb#diff-03f941e4ceb2839bfe0b49a5cc913e04919560e289603ae7b08b875f86ec568dR188